modules/ROOT/pages/6.1-release-notes.adoc

```diff
@@ -0,0 +1,1121 @@
+
+Previously, when the insertion point was within a `blockquote` element, and the `blockquote` element was the only block element in the {productname} document, there was no way of exiting the element without deleting all the text also within the element.
+
+As of this update, the `end_container_on_empty_block` has been changed such that pressing the *Return* key twice will exit the insertion point from a `blockquote` element. This is the same action that exits other block elements, such as lists.
+
+
+=== The default value for `end_container_on_empty_block` option has been changed to `blockquote`
+
+Previously, the `end_container_on_empty_block` took a boolean value and, by default, that value was `false`.
+
+As part of the changes made to enable the _pressing Enter or Return twice exits an in-document `blockquote` element_ change noted above, this option’s default value has been changed.
+
+As of {productname} 6.1, `end_container_on_empty_block` takes the value `blockquote` by default.
+
+NOTE: The boolean values, `true` and `false` remain valid values to assign to this option.
+
+
+=== Custom elements are now treated as non-empty elements by the schema
+
+Previously {productname} treated custom elements as empty elements. The {productname} schema was such that the `isEmpty` check returned `true` for all custom elements.
+
+With this update the {productname} schema has been updated; the `isEmpty` check now returns `false` for custom elements. 
+
+
+=== The `autocompleter` menu element is now positioned instead of the wrapper
+
+The `autocompleter` menu element — made available to a {productname} instance by the `editor.ui.registry.addAutocompleter()` function — presents a menu of formatting options when the `+/+` key is pressed in an active editor.
+
+If the current viewport is too small to present the entire menu, access to the ‘overflow’ is ensured by presenting a scrollbar in the menu.
+
+Previously, however, the rendering of this menu’s height (which, in turn, establishes whether a scroll bar is required) was done with the wrong element. Rendering was done by `tox-autocompleter`, not `tox-menu`. (`tox-menu` renders the height of all other {productname} menus.)
+
+In some circumstances this error resulted in the top or bottom menu borders to scroll out of view, making the meny appear unusable.
+
+This update corrects this. Rendering of the `autocompleter` menu height is now assigned to `tox-menu` and its menu borders do not scroll out of view.
+
+
+=== Link menu and toolbar buttons now always execute the `mceLink` command
+
+Selecting *Insert > Link…* or clicking the Link toolbar button did not, previously, execute the `mceLink` command. Equivalently, if the `link_quicklink` option was enabled, using the Ctrl+K or ⌘+K shortcut to open the Contextual Link Toolbar did not execute `mceLink` either.
```

Comment:
The second sentence here is incorrect... previously the shortcuts did use the `mceLink` command, it was only the UI elements that did not.

Reply:
Corrected the second sentence.